### ISSUE TITLE
Missing TF_VAR documentation

### ISSUE THREAD

**Issue by rainleander** [equinix/terraform-metal-openstack#58]
README.md needs to include:

export TF_VAR_metal_create_project=false

OR the default needs to include this variable.

**Comment by displague** [comment]
The sample files should be updated to include a snippet like this:

```
# Use an existing project:
# metal_create_project        = false
# metal_project_id           = "..."
#
# Or create a new one (default), requiring organization_id :
# metal_create_project        = true
metal_organization_id       = "..."
```

The organization id should be prompted for, but it is not because the organization_id has a default of "null".  The organization_id is not needed if the project_id is provided. 

Is there a way to require one of these two variables to be set that allows for Terraform to prompt for one or both values, or otherwise fail if neither value is set? Perhaps we can create a variable (local?) with a validation pattern that will fail with a meaningful error if organization or project id are not set.

**Comment by displague** [comment]
The sample files should also be updated to use the latest models and facilities.

**Comment by displague** [comment]
This was resolved by #59.
Metros are being introduced in #63
